client/views/helpers/chunks/question_div/question_div.html

```diff
@@ -62,7 +62,7 @@
       </div>
       {{#if adminButtons}}
         <div class="question-highlight">
-          <input class="remote-highlight" type="checkbox" name="highlight" value="0"> Highlight the question
+          <label class="remote-highlight"><input type="checkbox" name="highlight" value="0">Highlight the question</label>
```

Comment:
@singhsanket143 now the label is a little bit off. Can you add some space between the label and the checkbox and make the label vertically centered? Also, add make the cursor be `pointer` on hover.

Reply:
@peter-hank I have made the corresponding changes